js-api/src/chem.ts

```diff
@@ -42,6 +42,10 @@ export namespace chem {
   export const STORAGE_NAME = 'sketcher';
   export const KEY = 'selected';
 
+  export interface IValidationRes {
```

Comment:
It's ok to fully spell it, as it's not going to be used extensively in the code

Reply:
Also in the core we use simpler notation, validation is typically a function that returns either error description as a string, or null if everything is correct

Reply:
Changed function returne type to string | null